src/Main.java

```diff
@@ -1,5 +1,77 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //Задача 1
+        int age = 10;
+        if (age >= 18) {
+            System.out.println("Совершеннолетний");
+        } else {
+            System.out.println("Несовершеннолетний");
+        }
+        //Задача 2
+        int temper = 6;
+        if (temper < 5) {
+            System.out.println("На улице " + temper + " градусов,нужно надеть шапку");
+        }
+        if (temper > 5) {
+            System.out.println("На улице " + temper + " градусов,можно идти без шапки");
+        }
+        //Задача 3
+        int speed = 50;
+        if (speed > 60) {
+            System.out.println("Если скорость " + speed + " то придется заплатить штраф");
+        }
+        if (speed < 60) {
+            System.out.println("Если скорость " + speed + " то можно ездить спокойно");
+        }
+        //Задача 4
+        int agee = 50;
+        if (agee >= 2 && agee <= 6) {
+            System.out.println("Если возраст человека равен " + agee + ", то ему нужно ходить в десткий сад");
+        } else if (agee > 6 && agee < 18) {
+            System.out.println("Если возраст человека равен " + agee + ", то ему нужно ходить в школу");
+        } else if (agee >= 18 && agee <= 24) {
+            System.out.println("Если возраст человека равен " + agee + ", то ему нужно ходить в университет");
+        } else if (agee > 24) {
+            System.out.println("Если возраст человека равен " + agee + ", то ему пора ходить на работу");
+        }
+        //Задача 5
+        int childAge = 15;
```

Comment:
```suggestion
        age = 15;
```
зачем заводить новую переменную, когда можно просто переиспользовать уже объявленную выше?